src/main/java/com/google/sps/data/Constants.java

```diff
@@ -25,7 +25,7 @@
     public static final String COMMENT_ENTITY_TYPE = "Comment";
     public static final String COMMENT_NAME = "Nick Name";
     public static final String COMMENT_MSG = "Message";
-    public static final String CIVIC_API_KEY = "CIVIC_API_KEY";
+    public static final String CIVIC_API_KEY = "AIzaSyAHQRFZpZBZSOeUI-4pmfoHV4SM0eBSlS4";
```

Comment:
this shouldn't be here ! maybe add to git ignore

Reply:
I think you should try to do this like Nikita did with environment variables!

Reply:
This was the work-around I had to use since the environment variables that nikita showed me how to use didn't work properly. Maybe I messed it up but this was the only way I could get the api to load locally.